pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -63,10 +62,11 @@ type AbstractStorageClass struct {
 	makeStoragePolicyAPI func(ctx context.Context, connection *vclib.VSphereConnection, infra *v1.Infrastructure) vCenterInterface
 	scStateEvaluator     *csiscc.StorageClassStateEvaluator
 
-	policyName string
-	backoff    wait.Backoff
-	nextCheck  time.Time
-	lastCheck  time.Time
+	policyName      string
```

Comment:
nit: rename `policyName` to `sharedPolicyName`, to make it obvious that the same policy name is used for both vCenters.